shell/utils/validators/formRules/__tests__/index.test.ts

```diff
@@ -1117,6 +1117,7 @@ describe('formRules', () => {
     const testCases = [
       ['', undefined],
       ['https://github.com/rancher/fleet-examples.git', undefined],
+      ['oci://test.rancher.io/charts/mychart', undefined],
```

Comment:
can this be expanded to cover all types defined in https://ranchermanager.docs.rancher.com/v2.9/how-to-guides/new-user-guides/helm-charts-in-rancher/oci-repositories#add-an-oci-based-helm-chart-repository